### DISCUSSION TITLE
"share" operator

### DISCUSSION THREAD

**Discussion by tsouza** [renggli/dart-rx#20] [Q&A] [answered]
I think the answer is `no`, but I would I like to confirm. Is there any combination of available operators that would result in the same behaviour as the [share](https://github.com/ReactiveX/rxjs/blob/master/src/internal/operators/share.ts) operator?

If not, I'll send a PR

**Comment by renggli** [comment, marked as answer]
Not available as a operator itself, but can by done by using `observable.multicast().refCount()`, see test here: https://github.com/renggli/dart-rx/blob/68bddf39dc44e7cbcf058cef4064f2a075d09e17/test/operators_test.dart#L1721-L1754
